packages/cli/src/commands/rewards/slasher.ts

```diff
@@ -0,0 +1,108 @@
+import { Address } from '@celo/contractkit/lib/base'
+import { ValidatorsWrapper } from '@celo/contractkit/lib/wrappers/Validators'
+import { bitIsSet, parseBlockExtraData } from '@celo/utils/lib/istanbul'
+import { BaseCommand } from '../../base'
+
+export default class Slasher extends BaseCommand {
+  static description = 'Mines for slashable downtime'
+
+  static flags = {
+    ...BaseCommand.flags,
+  }
+
+  static args = []
+
+  static examples = ['slasher']
+
+  async run() {
```

Comment:
This whole commands looks like a `slasher bot`. In that case, i would recommend extracting it outside. Probably to contractKit. and then call it from here.

With that i will be more reusable. And also I'm of the opinion that cli shouldn't be that intelligent, instead it should call others to do the job for them